src/api/app/views/webui/package/_buildstatus.html.haml

```diff
@@ -0,0 +1,41 @@
+- unless @excluded_results_count.zero? && !@show_excluded
+  .show_excluded_box
+    = check_box_tag :show_excluded, true, @show_excluded
+    - label_message = @excluded_results_count.zero? ? 'Hide' :  "Show #{@excluded_results_count}"
+    = label_tag :show_excluded, "#{label_message} excluded results"
+
+- @buildresults.each_pair do |package, results|
+  %h3= package
+  #package-buildstatus
+    %table
+      - if results.present?
+        - previous_repo = nil
```

Comment:
Out of curiosity: why do we need to save the previous repository and compare it with the current?

Reply:
IIRC, is for not repeat the repository in each architecture.